api-editor/gui/src/features/packageData/selectionView/StatisticsView.tsx

```diff
@@ -0,0 +1,217 @@
+    const usedThreshold = 1;
+
+    const classLabels = ['full', 'public', 'used'];
+    const classValues = getClassValues(pythonPackage, usages, usedThreshold);
+    const classBarChart = createBarChart(classLabels, classValues, 'Classes');
+
+    const functionLabels = ['full', 'public', 'used'];
+    const functionValues = getFunctionValues(pythonPackage, usages, usedThreshold);
+    const functionBarChart = createBarChart(functionLabels, functionValues, 'Functions');
+
+    const parameterLabels = ['full', 'public', 'used', 'useful'];
+    const parameterValues = getParameterValues(pythonPackage, usages, usedThreshold);
+    const parameterBarChart = createBarChart(parameterLabels, parameterValues, 'Parameters');
+
+    const thresholds = [...Array(26).keys()];
+    thresholds.shift();
+    const classLineChart = createLineChart(usages, thresholds, getUsedClasses, 'Classes per Threshold');
+    const functionLineChart = createLineChart(usages, thresholds, getUsedFunctions, 'Classes per Threshold');
+    const parameterLineChart = createLineChart(usages, thresholds, getUsefulParameters, 'Classes per Threshold');
+
+    return (
+        <VStack>
+            {classBarChart},{functionBarChart},{parameterBarChart},{classLineChart},{functionLineChart},
+            {parameterLineChart}
+        </VStack>
+    );
+};
+
+let getClassValues = function (pythonPackage: PythonPackage, usages: UsageCountStore, usedThreshold: number) {
+    const classes = pythonPackage.getClasses();
+    let result: number[] = [];
+
+    let publicClasses = 0;
+    classes.forEach((element) => {
+        publicClasses += element.isPublic ? 1 : 0;
+    });
+
+    result.push(classes.length);
+    result.push(publicClasses);
+    result.push(getUsedClasses(usages, usedThreshold));
+
+    return result;
+};
+
+let getFunctionValues = function (pythonPackage: PythonPackage, usages: UsageCountStore, usedThreshold: number) {
+    const functions = pythonPackage.getFunctions();
+    let result: number[] = [];
+
+    let publicFunctions = 0;
+    functions.forEach((element) => {
+        publicFunctions += element.isPublic ? 1 : 0;
+    });
+
+    result.push(functions.length);
+    result.push(publicFunctions);
+    result.push(getUsedFunctions(usages, usedThreshold));
+
+    return result;
+};
+
+let getParameterValues = function (pythonPackage: PythonPackage, usages: UsageCountStore, usedThreshold: number) {
+    const parameters = pythonPackage.getParameter();
+    let result: number[] = [];
+
+    let publicParameters = 0;
+    parameters.forEach((element) => {
+        publicParameters += element.isPublic ? 1 : 0;
+    });
+
+    result.push(parameters.length);
+    result.push(publicParameters);
+    result.push(getUsedParameters(usages, usedThreshold).length);
+    result.push(getUsefulParameters(usages, usedThreshold));
+
+    return result;
+};
+
+let getUsedClasses = function (usages: UsageCountStore, usedThreshold: number): number {
```

Comment:
Should only count public used classes.

Reply:
Also count move this to the `UsageCountStore`.

Reply:
done